todo.go

```diff
@@ -20,7 +20,7 @@ func todo(id int, all bool) {
 	check(err)
 
 	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"title", "format", "tags", "reason", "url"})
+	table.SetHeader([]string{"title", "format", "tags", "reason", "updated", "url"})
```

Comment:
sgtm, but drop the reason field, we have no proposals that don't have reviews now.